src/client-cohttp-eio/opentelemetry_client_cohttp_eio.mli

```diff
@@ -43,7 +44,7 @@ val with_setup :
   ?stop:bool Atomic.t ->
   ?config:Config.t ->
   ?enable:bool ->
-  (Eio_unix.Stdenv.base -> 'a) ->
+  (unit -> 'a) ->
```

Comment:
This change is because we should not *force* the computation to take the Eio stdenv. If users want to provide their computations with the environment, they can do so via a partial application or whatever.